pkg/skaffold/runner/v1/dev.go

```diff
@@ -226,6 +228,22 @@ func (r *SkaffoldRunner) Dev(ctx context.Context, out io.Writer, artifacts []*la
 			continue
 		}
 
+		var absMatches []string
+		// Add absolute typed rules
+		if artifact.Sync != nil {
+			for _, pt := range artifact.Sync.Manual {
+				if pt.Type == "absolute" {
+					// list files and add to dependencies list
+					matches, err := doublestar.Glob(pt.Src)
+					if err != nil {
+						return fmt.Errorf("pattern error for %q: %w", filepath.Dir(pt.Src), err)
```

Comment:
Would the error be more readable if pt.Src (vs filepath.Dir(pt.Src) ) was used?  I think it might be easier for users to interpret the error if the pattern was in the error (vs the Dir).